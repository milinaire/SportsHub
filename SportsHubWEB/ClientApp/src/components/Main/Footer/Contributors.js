import React, { Component } from "react";
import "./Footer.css";

export class Contributors extends Component {
  state = {
    Headline: "some headline from db",
    Text: "text from db",
  };
  componentDidMount(){
    this.props.setLayout('base')
  }
  componentWillUnmount(){
    this.props.setLayout('home')
  }
  render() {
    return (
      <div>
        {this.props.match.params.name}
        <h1>{this.state.Headline}</h1>
        <p>{this.state.Text}</p>
      </div>
    );
  }
}
