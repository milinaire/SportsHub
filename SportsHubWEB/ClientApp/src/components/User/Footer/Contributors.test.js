
// Auto-generated do not edit


/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import {Contributors} from "./Contributors";




describe('Main\Footer\Contributors test', () => {
  it('Main\Footer\Contributors should match snapshot', () => {
    const component = renderer.create(<Contributors/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
