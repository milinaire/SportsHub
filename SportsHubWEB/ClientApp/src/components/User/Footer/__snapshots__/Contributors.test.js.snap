// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainFooterContributors test MainFooterContributors should match snapshot 1`] = `
<div
  className="main-part-footer"
>
  <h1>
    some headline from db
  </h1>
  <p>
    text from db
  </p>
</div>
`;
